<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Selection</class>
 <widget class="QDockWidget" name="Selection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selection</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Selection &amp;Type:</string>
          </property>
          <property name="buddy">
           <cstring>Selection_type_combo_box</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Selection_type_combo_box">
          <item>
           <property name="text">
            <string>Vertex</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Facet</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Edge</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="Insertion_radio_button">
               <property name="text">
                <string>Insertion</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="Removal_radio_button">
               <property name="text">
                <string>Removal</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Brush &amp;size:</string>
                 </property>
                 <property name="buddy">
                  <cstring>Brush_size_spin_box</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="Brush_size_spin_box"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="Select_all_button">
             <property name="text">
              <string>Select &amp;All</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Clear_button">
             <property name="text">
              <string>&amp;Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Isolated &amp;Component Size:</string>
             </property>
             <property name="buddy">
              <cstring>Threshold_size_spin_box</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="Threshold_size_spin_box">
             <property name="maximum">
              <number>999999999</number>
             </property>
             <property name="value">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Get_minimum_button">
             <property name="text">
              <string>&amp;Get Minimum</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="Select_isolated_components_button">
           <property name="text">
            <string>Select &amp;Isolated Components Below Threshold</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Dilate or erode selection:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="Dilate_erode_spin_box">
         <property name="minimum">
          <number>-50</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Dilate_erode_button">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Create_point_set_item_button">
      <property name="text">
       <string>Create Point Set Item from Selected Vertices</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Create_polyline_item_button">
      <property name="text">
       <string>Create Polyline Item from Selected Edges</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Create_polyhedron_item_button">
      <property name="text">
       <string>Create Polyhedron Item from Selected Facets</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Erase_selected_facets_button">
      <property name="text">
       <string>Erase Selected Facets from Polyhedron Item</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Create_selection_item_button">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Create Selection Item</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
